rllib/policy/policy.py

```diff
@@ -740,8 +740,44 @@ def get_state(self) -> PolicyState:
             # The current global timestep.
             "global_timestep": self.global_timestep,
         }
+        if self.config.get("enable_connectors", False):
+            connector_configs = {}
+            if self.agent_connectors:
+                connector_configs["agent"] = self.agent_connectors.to_config()
+            if self.action_connectors:
+                connector_configs["action"] = self.action_connectors.to_config()
+            state["connector_configs"] = connector_configs
         return state
 
+    @ExperimentalAPI
```

Comment:
Should we use `PublicApi(alpha)` here as well?

Reply:
👍